ds_judgements_public_ui/sass/includes/_judgment_document_paragraph_anchors.scss

```diff
@@ -7,7 +7,8 @@
     }
 
     &:hover {
-      .judgment-body__copy-link {
+      .judgment-body__copy-link,
+      .judgment-body__copy-tooltip {
```

Comment:
Should this one be helptext? We've search-replaced tooltip elsewhere.
(Might also be worth a code search for tooltip to see if we're missing any)

Reply:
This one is correct. Basically we have tooltip and helptext. Helptext in the "icon" variation is just text that explains what to do. The "tooltip" version is the thing you actually have to click.